src/pages/Home.css

```diff
@@ -4,24 +4,23 @@ h1 {
     font-family: 'Spectral', serif;
     display: flex;
     justify-content: center;
-    color: white;
+    color: black;
     font-size: 3rem;
 }
-
-.cashiswineImgHome {
-    display: flex;
-    margin: auto;
-    width: 250px;
-    height: auto;
-    margin-bottom: 1rem;
-}
 .background {
     display: flex;
     align-items: center;
-    justify-content: center;
-    text-align: center;
     position: relative;
-    height: 35vh;
+    height: 40vh;
+    z-index: 2;
+}
+.backgroundh1 {
+    display: flex;
+    background-color: white;
+    margin-left: 10rem;
```

Comment:
you should use no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead